External/FEXCore/Source/Interface/Core/JIT/Arm64/JITClass.h

```diff
@@ -350,6 +350,7 @@ class JITCore final : public CPUBackend, public vixl::aarch64::Assembler  {
   DEF_OP(FillRegister);
   DEF_OP(LoadFlag);
   DEF_OP(StoreFlag);
+  MemOperand GenerateMemOperand(uint8_t AccessSize, aarch64::Register Base, IR::OrderedNodeWrapper Offset, IR::MemOffsetType OffsetType, uint8_t OffsetScale);
```

Comment:
Move this above the IR handler definitions for cleanliness.

Reply:
Done